CMakeLists.txt

```diff
@@ -155,6 +173,9 @@ endif()
 function(halide_project name folder)
   add_executable("${name}" ${ARGN})
   if (MSVC)
+    if(NOT HALIDE_ENABLE_RTTI)
```

Comment:
It's a little surprising that CMake doesn't offer a standard option for 'enable/disable rtti'. (Maybe it does and I haven't checked?)

Reply:
It does not. I think CMake tries to have just enough built-ins to get by, which is fair considering the number of compilers and platforms they support.